livestream-compose-sample/src/main/java/io/getstream/livestream/compose/LiveStreamScreen.kt

```diff
@@ -0,0 +1,142 @@
+import io.getstream.chat.android.compose.ui.messages.header.MessageListHeader
+import io.getstream.chat.android.compose.ui.messages.list.MessageList
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.chat.android.compose.viewmodel.messages.MessageComposerViewModel
+import io.getstream.chat.android.compose.viewmodel.messages.MessageListViewModel
+
+
+@Composable
+fun LiveStreamScreen(
+    composerViewModel: MessageComposerViewModel,
+    listViewModel: MessageListViewModel,
+    onBackPressed: () -> Unit = {}
+) {
+    val currentState = listViewModel.currentMessagesState
+    val messageMode = listViewModel.messageMode
+    val isNetworkAvailable by listViewModel.isOnline.collectAsState()
+    val user by listViewModel.user.collectAsState()
+    LaunchedEffect(Unit) {
+        listViewModel.start()
+    }
+
+    val backAction = {
+        val isInThread = listViewModel.isInThread
+        val isShowingOverlay = listViewModel.isShowingOverlay
+
+        when {
+            isShowingOverlay -> listViewModel.selectMessage(null)
+            isInThread -> {
+                listViewModel.leaveThread()
+                composerViewModel.leaveThread()
+            }
+            else -> onBackPressed()
+        }
+    }
+
+    SystemBackPressedHandler(isEnabled = true, onBackPressed = backAction)
+
+    ChatTheme(isInDarkMode = true) {
+        Box(modifier = Modifier.fillMaxSize()) {
+            Scaffold(
+                modifier = Modifier.fillMaxSize(),
+                topBar = {
+                    MessageListHeader(
+                        modifier = Modifier
+                            .height(56.dp),
+                        channel = listViewModel.channel,
+                        currentUser = user,
+                        isNetworkAvailable = isNetworkAvailable,
+                        messageMode = messageMode,
+                        onBackPressed = backAction,
+                        onHeaderActionClick = {}
+                    )
+                },
+                bottomBar = {
+                    MyCustomComposer(composerViewModel)
+                }
+            ) {
+                MessageList(
+                    modifier = Modifier
+                        .padding(it)
+                        .fillMaxSize()
+                        .background(Color.Transparent),
+                    viewModel = listViewModel,
+                    onThreadClick = { message ->
+                        composerViewModel.setMessageMode(Thread(message))
+                        listViewModel.openMessageThread(message)
+                    }
+                )
+                YTPlayer()
+            }
+        }
+    }
+}
+
+@Composable
+fun MyCustomComposer(composerViewModel: MessageComposerViewModel) {
+    MessageComposer(
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight(),
+        viewModel = composerViewModel,
+        integrations = {},
+        input = {
```

Comment:
As I mentioned in our DM, we don't need to customize the message Input. The input is fine, it's only the integrations we need to remove.

I do understand it's a huge learning curve to grasp Compose and our components, but please try to avoid C/P content from our guides/other samples, as I think this is counter-productive.

The point of these showcases is not to just build something that looks like a Livestream app. The point is to see if our SDK can naturally and intuitively help us do this. And to see if the API is clean and intuitive enough for someone who hasn't had a lot of Compose experience (like you) to consume it and build something like this. C/P effectively negates this goal of our apps and we don't know if the API is well designed if we're just reusing code that I've written, with a full understanding our of SDK.